test.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        """This is a docstring.
</pre><p>Are you sure?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DanWertheimer/reviewnb-test/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>